mediaflow_proxy/utils/crypto_utils.py

```
@@ -1,151 +1,151 @@
+            if "exp" in data:
+                if data["exp"] < time.time():
+                    raise HTTPException(status_code=401, detail="Token has expired")
+                del data["exp"]  # Remove expiration from the data
+
+            if "ip" in data:
+                if data["ip"] != client_ip:
+                    raise HTTPException(status_code=403, detail="IP address mismatch")
+                del data["ip"]  # Remove IP from the data
+
+            return data
+        except Exception as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)